sentence_transformers/model_card.py

```diff
@@ -283,6 +283,7 @@ class SentenceTransformerModelCardData(CardData):
             "sentence-transformers",
             "sentence-similarity",
             "feature-extraction",
+            "generated_from_trainer",
```

Comment:
The model card is only generated if:
1. the SentenceTransformerTrainer has been initialized with this model, or
2. the loaded model wasn't a Sentence Transformer model

Otherwise, the original model card gets reused. E.g. if you load and save/upload BAAI/bge-base-en-v1.5 without training then it'll reuse that model card.

So, we would only get `generated_with_trainer` "wrongly" if someone loads a non-Sentence Transformer model, doesn't train it, and then uploads it. That said, I think this is really easy to fix, so I'll change it.

Reply:
Thanks for clarifying!

Reply:
Resolved this via 10d70ff48a5a9e88603f14ef859fb1452c01f560